Add details to the enhancement and fix section of the 1.3 release (#236)
KxSystems/kx-vscode: CHANGELOG.md

All notable changes to the kdb extension will be documented in this file.

# v1.3.0

### Enhancements

> - Ability to connect to kdb Insights Enterprise using a WSL connection.

Reviewer: and SSH

Author: @ecmel how do i word this?
- Ability to connect to kdb Insights Enterprise using a WSL and SSH connection.

or 

- Ability to connect to kdb Insights Enterprise using a WSL connection and SSH.

or something different

Reviewer: @LizNorris Ability to connect to Insights Enterprise in case of remote development environments such as WSL and SSH.

(Note: We possibly support all kind of remote dev since we use the vscode apis for login now)

Author: ```suggestion
- Ability to connect to Insights Enterprise in case of remote development environments such as WSL and SSH.
```

- Execute files from the right-click menu.

> 

Reviewer 2: on Results tab, hover the mouse at the column, show the column Q type if it exist

Reviewer 2: only for insights connections

Author: - On the Results tab, on a kdb Insights Enterprise connection, hover the mouse over a column header to see  the column Q type if it exists.

Author: ```suggestion
- On the Results tab, on a kdb Insights Enterprise connection, hover the mouse over a column header to see the column Q type if it exists.
```

- Linting enhancements:
    - Implement the following rules:
        - Declare after use
        - Invalid Escape
        - Fixed Seed
        - Too many arguments - when more than 8 arguments are being passed

>     - Line length limits have been removed to ensure there aer no warnings after a long block comment.

Reviewer 3: are*

Author: ```suggestion
    - Line length limits have been removed to ensure there are no warnings after a long block comment.
```

    - Handle .z.exit correctly.
    - Optionally disable linting.

### Fixes

- Ability to switch users connected to a kdb Insights Enterprise URL. The new flow to switch users when you are already logged in is as follows:
    - Disconnect from the URL.
    - Log into the URL using browser and log out of environment.
    - On reconnecting you are asked to enter your login details and you can chose a different user.

- Fix to ensure you can login again if a login attempt fails.

- Fix for an invalid "settings.json" file causing "Add Connection" to fail without error.

- Ensure the "Execute Entire File" button works even if the cursor is not in the code editor window.

- Fixes for the Data Sources:

>     - Custom APIs are nolonger listed, these will be added in a future release when the execution of a Custom API is supported.

Reviewer 3: no longer*

    - The "Run" button will be greyed out while a Data Source is executing, to ensure there are no concurrent executions.
    - To see Data Source results within the "Output" tab, ensure that the output is from "q Console Output".

- Fixes for the tree:

>     - The only variables being displayed were longs 

Reviewer 3: Ensure that all sentences end in "."
